cli/src/common_utils/utility_scripts.sh

```diff
@@ -142,7 +146,12 @@ function ensure_healthy_cluster {
 
                 if [[ ${attempt} -ge 20 ]]; then
                     _lerror "Giving up after ${attempt} attempts."
-                    exit 20
+                    if [[ "$_exit_on_err" == true ]]; then
+                        exit 20
```

Comment:
Here rather than exit can we reset counter and run `pcs resource cleanup --all`?

Reply:
@ypise, in that case I think update will be affected, lets not do anything in this function, let the callee take the decision to exit or to continue.